traffic_portal/app/src/common/modules/navigation/NavigationController.js

```diff
@@ -21,7 +21,7 @@ var NavigationController = function($scope, $log, $state, $location, $window, $t
 
     $scope.appName = propertiesModel.properties.name;
 
-    $scope.isProd = propertiesModel.properties.environment.isProd;
+    $scope.isProd = (propertiesModel.properties.environment) ? propertiesModel.properties.environment.isProd : false;
```

Comment:
Should this safety check be for `propertiesModel.properties.environment.isProd`?

Reply:
see prev comment.